### PR TITLE
Source Replacement 2.0

### PR DESCRIPTION
**Pull request by ReiiYuki**
### Why

* Enhance ability of source replacement work with vConsole
* Blocking the race condition of script executing during the replacement process
* Refactor and improve test quality

### CODE REVIEW

**Comment by clowiie** [comment]
LGTM Krub

**Review by clowiie — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -1,10 +1,9 @@
 {
   "name": "source-replacement",
-  "version": "0.0.0",
-  "main": "build/index.js",
-  "entry": "src/index.ts",
+  "version": "2.0.0",
```
Comment (nitpum) [inline package.json:3]: Is this not bump by semantic-release?

File: src/executors/code-blocker/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,13 @@
+describe('code-blocker', () => {
```
Comment (nitpum) [inline src/executors/code-blocker/index.test.ts:1]:
```suggestion
describe('code-blocker()', () => {
```

File: src/executors/source-replacement/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,19 @@
+describe('source-replacement', () => {
```
Comment (nitpum) [inline src/executors/source-replacement/index.test.ts:1]:
```suggestion
describe('source-replacement()', () => {
```

File: src/core/preventPerformExistingScriptDuringInjection/index.test.ts [hunk 4]
```diff
@@ -0,0 +1,81 @@
+import { REPLACEMENT_SOURCE_KEY } from 'debugConstants'
+import preventPerformExistingScriptDuringInjection from '.'
+
+describe('preventPerformExistingScriptDuringInjection', () => {
```
Comment (nitpum) [inline src/core/preventPerformExistingScriptDuringInjection/index.test.ts]:
```suggestion
describe('preventPerformExistingScriptDuringInjection()', () => {
```

**Review by nitpum — APPROVED** [review]

File: src/executors/code-blocker/index.test.ts (hunk 2 above)
Comment (ReiiYuki) [reply src/executors/code-blocker/index.test.ts:1]: This executor does not export as function

File: rollup.config.js [hunk 5]
```diff
@@ -1,34 +1,45 @@
...
+    exclude: 'src/**/*.test.ts',
+  }),
+  json(),
+  visualizer({
+    filename: path.resolve(config.deploy, 'stat.html'),
+  }),
+]
+
+export default [
+  {
```
Comment (nitpum) [inline rollup.config.js]: Indent should be tab instead of space

File: src/executors/source-replacement/index.test.ts (hunk 3 above)
Comment (ReiiYuki) [reply src/executors/source-replacement/index.test.ts:1]: This executor does not export as function

File: package.json (hunk 1 above)
Comment (ReiiYuki) [reply package.json:3]: www previous set up it not auto bump version, so we need to manual versioning for this build

**Comment by nitpum** [comment]
Don't forget to add `BREAKING CHANGE:` to footer

**Pull request merged**